torch/testing/_internal/common_utils.py

```diff
@@ -2170,6 +2170,9 @@ def assertEqual(
         # Hide this function from `pytest`'s traceback
         __tracebackhide__ = True
 
+        # TODO: the Tensor compare uses bunch of operations which is currently not
```

Comment:
Great comment -- bonus points if we file an issue for MPS-related test improvements and link to it from here, but not a requirement

Reply:
Done: https://github.com/pytorch/pytorch/issues/77144